presto-main/src/test/java/com/facebook/presto/sql/planner/optimizations/TestReorderJoins.java

```diff
@@ -33,7 +33,9 @@
 {
     public TestReorderJoins()
     {
-        super(ImmutableMap.of(SystemSessionProperties.JOIN_REORDERING_STRATEGY, "COST_BASED"));
+        super(ImmutableMap.of(
+                SystemSessionProperties.JOIN_REORDERING_STRATEGY, "COST_BASED",
+                SystemSessionProperties.JOIN_DISTRIBUTION_TYPE, "AUTOMATIC"));
```

Comment:
If "automatic" then also "cost_based".
But I am not sure if that can be lowercase for enums. Can it?